app/templates/application.hbs

```diff
@@ -4,16 +4,13 @@
     (if this.showEnvironment (concat " - " this.environmentName))
   )
 }}
-
-<AuApp class="{{if this.showEnvironment 'au-c-app--environment'}}">
-  {{#if this.showEnvironment}}
-    <EnvironmentBanner
-      @applicationName="Contact- en Organisatiegegevens"
-      @environmentName={{this.environmentName}}
-      @skin={{this.environmentInfo.skin}}
-      @message='Dit is de <strong>{{this.environmentInfo.title}}</strong> van het Contact- en Organisatiegegevens met fictieve en testgegevens. De productieomgeving met de echte data vind je op <a href="https://contactgegevens.lokaalbestuur.vlaanderen.be/" title="Contact-Organisatiegegevens" rel="tag">https://contactgegevens.lokaalbestuur.vlaanderen.be</a>.'
-    />
-  {{/if}}
+<AuApp>
+  <EnvironmentBanner
+    @applicationName="Contact- en Organisatiegegevens"
+    @environmentName={{this.environmentName}}
+    @skin={{this.environmentInfo.skin}}
+    @message='Dit is de <strong>{{this.environmentInfo.title}}</strong> van het Contact- en Organisatiegegevens met fictieve en testgegevens. De productieomgeving met de echte data vind je op <a href="https://contactgegevens.lokaalbestuur.vlaanderen.be/" title="Contact-Organisatiegegevens" rel="tag">https://contactgegevens.lokaalbestuur.vlaanderen.be</a>.'
+  />
```

Comment:
**Question:** Is the if statement removed on purpose? Do we want to display the banner all the time?

Reply:
oh my fault, will add that